qiskit_metal/analyses/quantization/lom_core_analysis.py

```diff
@@ -0,0 +1,1267 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""LOM analysis based on https://arxiv.org/pdf/2103.10344.pdf
+"""
+# pylint: disable=invalid-name
+
+# monkey patch to temporarily mock h5py dependency required by scqubits, which conflicts with
+# geopandas
+import sys
+
+
+# pylint: disable=wrong-import-position
+# pylint: disable=too-few-public-methods
+class DummyH5py:
+
+    @property
+    def Group(self):
+        pass
+
+
+sys.modules['h5py'] = DummyH5py
+
+from collections import defaultdict, namedtuple
+from typing import Any, List, Dict, Tuple, DefaultDict, Sequence, Mapping, Union
+import argparse
+
+import numpy as np
+import pandas as pd
+import scqubits as scq
+from sympy import Matrix
+from scipy import optimize, integrate
+
+from pyEPR.calcs.convert import Convert
+from pyEPR.calcs.constants import e_el as ele, hbar
+
+from qiskit_metal.toolbox_python.utility_functions import get_all_args
+from qiskit_metal.analyses.em.cpw_calculations import guided_wavelength
+from qiskit_metal.analyses.hamiltonian.states_energies import extract_energies
+
+from qiskit_metal import logger
+
+BasisTransform = namedtuple(
+    'BasisTransform',
+    ['orig_node_basis', 'node_jj_basis', 'num_negative_nodes'])
+
+Operator = namedtuple('Operator', ['op', 'add_hc'])
+
+
+class CouplingType:
+    CAPACITIVE = 'CAPACITIVE'
+    INDUCTIVE = 'INDUCTIVE'
+
+
+MHzRad = 2 * np.pi * 1e6
+GHzRad = 2 * np.pi * 1e9
+
+NANO = 1e-9
+FEMTO = 1e-15
+ONE_OVER_FEMTO = 1e15
```

Comment:
This is fine here, probably should be at top level config and throughout the entire module, but could also be specific to this namespace

Reply:
moved constants to constants.py for the entire quantization module